.github/workflows/ruby.yml

```
@@ -0,0 +1,22 @@
+name: Ruby Build Test
+
+on: push
+
+jobs:
+  build-test:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby-version: [2.0.0, 2.1, 2.2, 2.3.0, jruby-9.2.17.0]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Ruby ${{ matrix.ruby-version }}
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: ${{ matrix.ruby-version }}
+          bundler-cache: true
+      - name: Install dependencies
+        run: bundle install
+      - name: Run tests
+        run: bundle exec rake test
```

ディレクトリ指定あっても良いですけど、公式やtravisに合わせておきましょ

```suggestion
        run: bundle exec rake
```